scripts/prettier/src/prettier-helpers.js

```diff
@@ -47,10 +47,18 @@ function runPrettier(files, config = {}) {
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31889&commitId=cc27a0ab414bb25c5c49f41141d4d800345d2c86&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>